### ISSUE TITLE
Unable to use OriginCaCertificate

### ISSUE THREAD

**Issue by ehhthing** [pulumi/pulumi-cloudflare#301]
### What happened?

I'm trying to generate origin ca certificates. As per [the documentation](https://www.pulumi.com/registry/packages/cloudflare/api-docs/origincacertificate/), `This resource requires you use your Origin CA Key as the api_user_service_key, in conjunction with an api_token or email and api_key.`. However, if I try to have both the `api_user_service_key` and one of `api_token` or (`email` and `api_key`), the provider refuses to validate the configuration.

### Steps to reproduce

1. Set the `api_user_service_key`, `email` and `api_key` as needed (I've tried both using the config and the environment variables.)
2. Try to create a `OriginCaCertificate` resource.
3. An error occurs:
```
  cloudflare:index:OriginCaCertificate (cf-origin-cert):
    error: could not validate provider configuration: 3 errors occurred:
        * Invalid combination of arguments: "api_key": only one of `api_key,api_token,api_user_service_key` can be specified, but `api_token,api_user_service_key` were specified.
        * Invalid combination of arguments: "api_user_service_key": only one of `api_key,api_token,api_user_service_key` can be specified, but `api_token,api_user_service_key` were specified.
        * Invalid combination of arguments: "api_token": only one of `api_key,api_token,api_user_service_key` can be specified, but `api_token,api_user_service_key` were specified.
```

### Expected Behavior

A certificate will be generated.

### Actual Behavior

An error occurred, preventing the provider from validating its config.

### Contributing

Vote on this issue by adding a 👍 reaction. 
To contribute a fix for this issue, leave a comment (and link to your pull request, if you've opened one already).

**Comment by ehhthing** [comment]
On further inspection this seems to be an upstream issue, which was fixed in [v3.25.0](https://github.com/cloudflare/terraform-provider-cloudflare/releases/tag/v3.25.0)

**Comment by guineveresaenger** [comment]
Hi @ehhthing - thank you for filing this issue, and for the followup! Let me see if I can cut an upgrade for you here :)
